.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on:
       - self-hosted
       - generic
-    if: (!startsWith(github.head_ref, 'renovate/') || !startsWith(github.head_ref, 'dependabot/'))
+    if: ${{ (!startsWith(github.head_ref, 'renovate/') || !startsWith(github.head_ref, 'dependabot/')) }}
```

Comment:
should it be `&&` here?

Reply:
no, dependabot or renovate and it's a not

Reply:
yes, should be `&&`

Reply:
fixed